wallet-new/test/unit/Wallet/Inductive/Cardano.hs

```diff
@@ -248,40 +247,32 @@ equivalentT activeWallet esk = \mkWallet w ->
     toCardano InductiveCtxt{..} fld a = do
         ma' <- catchTranslateErrors $ runIntT' inductiveCtxtInt $ int a
         case ma' of
-          Left err -> throwError EquivalenceNotChecked {
-              equivalenceNotCheckedName   = fld
-            , equivalenceNotCheckedReason = err
-            , equivalenceNotCheckedEvents = inductiveCtxtEvents
-            }
-          Right (a', _ic') ->
-            return a'
+          Left err -> throwError
+              $ EquivalenceNotChecked fld err inductiveCtxtEvents
+          Right (a', _ic') -> return a'
 
 data EquivalenceViolation h =
     -- | Cardano wallet and pure wallet are not equivalent
-    EquivalenceViolation {
-        -- | The property we were checking
-        equivalenceViolationName     :: Text
-
-        -- | Evidence (what was not the same?)
-      , equivalenceViolationEvidence :: EquivalenceViolationEvidence
-
-        -- | The events that led to the error
-      , equivalenceViolationEvents   :: OldestFirst [] (WalletEvent h Addr)
-      }
+    -- EquivalenceViolation
```

Comment:
We should probably use Haddock annotations here to document this constructor along with its fields.